ios/Classes/WorkmanagerPlugin.h

```diff
@@ -1,4 +1,7 @@
 #import <Flutter/Flutter.h>
 
 @interface WorkmanagerPlugin : NSObject<FlutterPlugin>
+
++ (void)registerTaskWithIdentifier:(NSString *) taskIdentifier;
```

Comment:
Please document this public API

Reply:
@ened Do you mean adding a comment to this method?

Reply:
Method doc comment was added.